src/ecs/components/EntityType.hpp

```diff
@@ -11,6 +11,14 @@ namespace ecs::component
 {
     /// @brief EntityType component
     struct EntityType {
+        enum Types {
```

Comment:
What is that ? why should we need an EntityType ? pls  explain @aurelenc

Reply:
Required to let clients know what entity the server creates